build/docker/publish/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcc:8 as build
```

Comment:
This is a fairly large container to start from.

Reply:
maybe, but it has almost everything that is needed for build